nir/src/main/scala/scala/scalanative/nir/Versions.scala

```diff
@@ -21,7 +21,7 @@ object Versions {
    * new version of the toolchain.
    */
   final val compat: Int   = 8
-  final val revision: Int = 10
+  final val revision: Int = 9
```

Comment:
Decreasing number is probably an artifact of rebases. Has to be `compat = 8, revision = 11`